src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -625,6 +883,7 @@ void cpu_convert(const void *srcPtr,
             size,
             interimPrc,
             dstPrc,
+            false,
```

Comment:
Maybe I am missing smt, but clamp is always set to false there. Why we need this clamp filed at all?

Reply:
because by default, cpu_convert impl use `clamp= false` to keep align with ngraph impl. But currently this pr try to keep these `clamp= true` impl  interfaces in case some special cases may need to use these impl in future(BTW, ngraph impl also keeps this kind of `clamp= true` impl interface).

Reply:
We shouldn't introduce code paths which are not supposed to be used. It complicates the code w/o any reason, increase binary size and requires further maintenance. 
`clamp=true` mode insdie ngraph is used only for `convert_from_f32_to_f16_with_clamp` routine, which required for Constants compression logic (FP32->FP16). 
If it will be required on the plugin level we can directly call `convert_from_f32_to_f16_with_clamp` for such cases, but for now it is not the case.

Reply:
Done, removed 'clamp=true' related code path from cpu_convert